projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+# Datagrid - Style API
+
+## CSS Variables
+
+| CSS Variable Name                                     | Description                                                                  |
+| ----------------------------------------------------- | -----------------------------------------------------------------------------|
+| --clr-datagrid-font-color                             | Changes the text color of datagrid.                                          |
+| --clr-datagrid-default-border-color                   | Changes the border color of datagrid when default.                           |
```

Comment:
Changes the default datagrid border color

Reply:
As per your suggestions, changed it.